source/text/label.ts

```diff
@@ -392,6 +406,21 @@ export class Label {
 
 export namespace Label {
 
+    export enum Elide {
```

Comment:
why is there `Elide` and `WordWrap`? Is any combination possible? (WordWrap seems unused)